### PR TITLE
vice, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: games-emulation/vice/vice-3.3.recipe [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	runConfigure --omit-dirs sbinDir ./configure \
+		--sbindir=$commandBinDir
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make bindist
+	mkdir -p $settingsDir/VICE $commandBinDir
+	mv $sourceDir/SDL2VICE-x86-haiku-3.3/x* \
```
Comment (Begasus) [inline games-emulation/vice/vice-3.3.recipe]: x86 probably needs some replacement variant

**Comment by Begasus** [comment]
Need to find out how to import the previous patch for the settingsDir

**Comment by scottmc** [comment]
@Begasus perhaps this is ok to merge what you have so far and then open a new issue to address the settingsDir issue?

**Comment by Begasus** [comment]
@scottmc I'll give it a another spin today to see if I'm still on par on this one, will merge if no changes (and mark "?" for the arches)

**Pull request merged**